suite-native/module-accounts-import/src/components/AccountImportLoader.tsx

```diff
@@ -1,41 +1,76 @@
-import React from 'react';
+/* eslint-disable global-require */
+import React, { useEffect } from 'react';
+import { Image } from 'react-native';
+import Animated, {
+    useAnimatedStyle,
+    withDelay,
+    withSequence,
+    withTiming,
+} from 'react-native-reanimated';
 
-import { Box, Text } from '@suite-native/atoms';
+import { Box, Loader, Text, VStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
-const fetchingStyle = prepareNativeStyle(() => ({
+type AccountImportLoaderProps = {
+    onLoad(): void;
+};
+
+const loaderContainerStyle = prepareNativeStyle(utils => ({
     flex: 1,
-    justifyContent: 'center',
+    justifyContent: 'space-between',
     alignItems: 'center',
+    marginTop: utils.spacings.medium * 10,
 }));
 
-const textStyle = prepareNativeStyle<{ opacity?: number }>((_, { opacity = 1.0 }) => ({
-    marginBottom: 13,
-    opacity,
+const imageStyle = prepareNativeStyle(() => ({ width: 283, height: 166 }));
```

Comment:
@Nodonisko Is it ok to set image dimensions as absolute values if it's png? I did it to avoid making the image too big on large tablet screens.

Reply:
I think it should be OK since pixels values in RN are not really absolute. Just test how it looks on very small devices iPhone SE vs very big iPad etc.